notebooks/224-distilbert-Sequence-Classification/224-distilbert-Sequence-Classification.ipynb

```diff
@@ -0,0 +1,416 @@
+      "Find more information about API v2.0 and IR v11 at https://docs.openvino.ai\n"
+     ]
+    }
+   ],
+   "source": [
+    "onnx_model_path = 'model.onnx'\n",
+    "MODEL_DIR = f\"model/\"\n",
+    "os.makedirs(MODEL_DIR, exist_ok=True)\n",
+    "\n",
+    "onnx_model_path = Path(MODEL_DIR)/onnx_model_path\n",
+    "\n",
+    "!mo --input_model $onnx_model_path --output_dir $MODEL_DIR --model_name $checkpoint --input input_ids,attention_mask --input_shape [1,512],[1,512] --data_type FP32"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "878fbd90",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ir_model_xml = (Path(MODEL_DIR)/checkpoint).with_suffix(\".xml\")\n",
+    "ir_model_bin = (Path(MODEL_DIR)/checkpoint).with_suffix(\".bin\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "27cc074e",
+   "metadata": {},
+   "source": [
+    "Creating the input and output features for the inference"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "026c9eb5",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "BATCH_SIZE = 1\n",
+    "MAX_SEQ_LENGTH = 512\n",
+    "\n",
+    "warnings.filterwarnings(\"ignore\")\n",
+    "\n",
+    "ie = IECore()\n",
+    "net_onnx = ie.read_network(model=ir_model_xml, weights=ir_model_bin)\n",
+    "exec_net_onnx = ie.load_network(network=net_onnx, device_name=\"CPU\")\n",
+    "\n",
+    "output_layer_onnx = next(iter(exec_net_onnx.outputs))\n",
+    "input_layer_onnx = next(iter(exec_net_onnx.input_info))\n",
+    "\n",
+    "# get input and output names of nodes\n",
+    "input_keys = list(exec_net_onnx.input_info)\n",
+    "output_keys = list(exec_net_onnx.outputs.keys())\n",
+    "\n",
+    "# get network input size\n",
+    "input_size = exec_net_onnx.input_info[input_keys[0]].input_data.shape[1]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "48eb25e2",
+   "metadata": {},
+   "source": [
+    "Defining a softmax function to extract the prediction from the output of the IR format."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "de01fccc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def softmax(x):\n",
+    "    e_x = np.exp(x - np.max(x))\n",
+    "    return e_x / e_x.sum() \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2e778507",
+   "metadata": {},
+   "source": [
+    "# Inference"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cd429d55",
+   "metadata": {},
+   "source": [
+    "Creating a generic inference function to read the input and infer the result into 2 classes: Positive or Negative."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "cc0c91a6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def infer(input_text):\n",
+    "    input_text = tokenizer(input_text, padding='max_length', max_length = 512, truncation=True, return_tensors=\"pt\")\n",
```

Comment:
do we really need such huge length size?
I think 128 should be pretty enough if you do not demonstrate model work on large texts.
Also, tokenizer is able to return numpy result if you change return_tensors value from `pt` to `np`, it is more native for openvino. If you wish to highlight that torch tensors are acceptable as input, probably some additional comments needed as highlighting for this behaviour

Reply:
input_text = tokenizer(input_text, padding='max_length', max_length = 128, truncation=True, return_tensors="np")
![image](https://user-images.githubusercontent.com/95271966/203707436-72ca2a97-7e99-4a98-a916-3853d5a74433.png)

Reply:
yeah, probably these changes also requires update mo command line